src/pages/clients-corporate-list/forms/ClientTemplateForm.js

```diff
@@ -96,54 +88,53 @@ const ClientTemplateForm = ({
     plantId: null,
     cellPhone: null,
     cellPhoneRepeat: null,
-    otp: null,
-  });
+    otp: null
+  })
 
   const formik = useFormik({
     initialValues,
     enableReinitialize: true,
     validateOnChange: true,
 
     // validateOnBlur: true,
-    validate: (values) => {
-      const errors = {};
+    validate: values => {
+      const errors = {}
 
-      const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+      const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
 
       if (values.email1 && !emailRegex.test(values.email1)) {
-        errors.email1 = "Invalid email format";
+        errors.email1 = 'Invalid email format'
       }
 
       if (values.email2 && !emailRegex.test(values.email2)) {
-        errors.email2 = "Invalid email format";
+        errors.email2 = 'Invalid email format'
       }
 
-      return errors;
+      return errors
     },
     validationSchema: yup.object({
-      reference:
-        referenceRequired && yup.string().required("This field is required"),
-      expiryDate: yup.string().required("This field is required"),
-      countryId: yup.string().required("This field is required"),
-      cityId: yup.string().required("This field is required"),
-      name1: yup.string().required("This field is required"),
-      name: yup.string().required("This field is required"),
-      nationalityId: yup.string().required("This field is required"),
-      cellPhone: yup.string().required("This field is required"),
-      capital: yup.string().required("This field is required"),
-      lgsId: yup.string().required("This field is required"),
-      industry: yup.string().required("This field is required"),
-      activityId: yup.string().required("This field is required"),
-      street1: yup.string().required("This field is required"),
-      phone: yup.string().required("This field is required"),
+      reference: referenceRequired && yup.string().required('This field is required'),
```

Comment:
please remove all the "this field is required' messages, when a field is mandatory it is enough to have a red border, no need for the message